test/uk/gov/hmrc/merchandiseinbaggage/controllers/PaymentCalculationControllerSpec.scala

```diff
@@ -67,6 +67,9 @@ class PaymentCalculationControllerSpec extends DeclarationJourneyControllerSpec
           case Import =>
             status(eventualResult) mustBe 200
 
+            result must include(messages("paymentCalculation.greenchannel.p1"))
+            result must include(messages("paymentCalculation.greenchannel.p2"))
+            result must include(messages("paymentCalculation.greenchannel.p3"))
```

Comment:
we must continue adding these assertions to the ContentSpecs instead of in the ControllerSpecs